test/parallel/test-crypto-pbkdf2.js

```diff
@@ -65,6 +65,26 @@ common.expectsError(
   }
 );
 
+common.expectsError(
+  () => crypto.pbkdf2Sync('password', 'salt', -1, 20, null),
+  {
+    code: 'ERR_OUT_OF_RANGE',
+    type: RangeError,
+    message: 'The "iterations" argument is out of range'
+  }
+);
+
+['str', null, undefined, [], {}].forEach((i) => {
```

Comment:
Small nit: please use a more descriptive variable name than `i`.

Reply:
Thank you for your review.
I renamed `i` to `notNumber`.